cmd/podman/buildfarm/prune.go

```diff
@@ -0,0 +1,40 @@
+package buildfarm
+
+import (
+	"github.com/containers/podman/v4/cmd/podman/common"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/spf13/cobra"
+)
+
+var (
+	buildfarmPruneDescription = `Prune untagged and optionally unused images from a farm`
```

Comment:
Can we break the `prune` functionality out of this PR?  As mentioned in the design doc, this needs some more grooming.  Also, let's try to break as many things out into smaller PRs as possible to avoid gigantic and hard to review PRs.